README.md

```diff
@@ -1 +1,12 @@
-# Hi Trev!
\ No newline at end of file
+# Hi Trev!
+## Welcome to your development playground repository!
+
+You should probably read up about Markdown, that is what any file with a `.md` file extension contains – like this one!
+
+[Read about markdown here!](https://guides.github.com/features/mastering-markdown/)
```

Comment:
And maybe just bookmark this page - nothing you need to memorize but nice to have a reference!